site-src/guides/index.md

```diff
@@ -297,12 +297,17 @@ A cluster with:
 
    Wait until the gateway is ready.
 
+   Depending on the type of model server you have deployed, you must update the model field in the request body accordingly:
+   - vLLM Simulator Model Server: `food-review-1`
+   - CPU-Based Model Server: `food-review-0` or `food-review-1`
+   - GPU-Based Model Server: TODO
```

Comment:
I don't find the `mode` value for the GPU one. Please help me

Reply:
adding an answer so you'll learn more about how it's being set - 
you can find it here:
https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/c90106c0421e671dd91fb8761855aa8d99e2656a/config/manifests/vllm/gpu-deployment.yaml#L257